helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml

```diff
@@ -11,6 +11,7 @@ metadata:
     release: {{ .Release.Name }}
 
 ---
+{{- if not (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "aerospike-cluster") }}
```

Comment:
Can you make this condition just for `ClusterRole`?
Like this,

```
{{- if not (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" "aerospike-cluster") }}
# Role
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: aerospike-cluster
  labels:
    app: {{ template "aerospike-cluster.commonName" . }}
    chart: {{ .Chart.Name }}
    release: {{ .Release.Name }}
rules:
- apiGroups:
  - ""
  resources:
  - pods
  - nodes
  - services
  verbs:
  - get
  - list
- apiGroups:
  - aerospike.com
  resources:
  - '*'
  verbs:
  - '*'
{{- end }}
```

And also change `ClusterRoleBinding` name to `aerospike-cluster-{{ .Release.Namespace }}`.
Like this,
```
# RoleBinding
kind: ClusterRoleBinding
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: aerospike-cluster-{{ .Release.Namespace }}
  labels:
    app: {{ template "aerospike-cluster.commonName" . }}
    chart: {{ .Chart.Name }}
    release: {{ .Release.Name }}
roleRef:
  kind: ClusterRole
  name: aerospike-cluster
  apiGroup: rbac.authorization.k8s.io
subjects:
- kind: ServiceAccount
  name: aerospike-cluster
  namespace: {{ .Release.Namespace }}
```